site/src/@ndlib/gatsby-theme-marble/components/Shared/ActionButtonGroup/DownloadButton/DownloadModalContent/DownloadCitation/index.js

```diff
@@ -21,18 +21,18 @@ const DownloadCitation = ({ iiifManifest }) => {
   )
   let biblio = ''
   const metadata = {}
-  if (iiifManifest.type === 'Collection') {
-    biblio = `${typy(iiifManifest, 'label.en[0]').safeString}, ${typy(iiifManifest, 'slug').safeString.replace('item/', '')}. ${typy(iiifManifest, 'provider[0].homepage[0].label.en[0]').safeString}, Hesburgh Libraries, University of Notre Dame, South Bend, IN. ${typy(site, 'siteMetadata.siteUrl').safeString}/${typy(iiifManifest, 'slug').safeString}`
-  } else if (typy(iiifManifest, 'provider[0].homepage[0]label.en[0]').isString) {
-    iiifManifest.metadata.map((md) => {
+  if (marbleItem.type === 'Collection') {
+    biblio = `${typy(marbleItem, 'label.en[0]').safeString}, ${typy(marbleItem, 'slug').safeString.replace('item/', '')}. ${typy(marbleItem, 'provider[0].homepage[0].label.en[0]').safeString}, Hesburgh Libraries, University of Notre Dame, South Bend, IN. ${typy(site, 'siteMetadata.siteUrl').safeString}/${typy(marbleItem, 'slug').safeString}`
```

Comment:
The metadata fields have changed shape with the data conversion so you'll need to remap them. For example `${typy(marbleItem, 'label.en[0]').safeString}` becomes `${typy(marbleItem, 'title').safeString}`.